Assets/BowAndArrowImproved/Scripts/UpdateScoreOnDeath.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using UnityEngine;
+
+public class UpdateScoreOnDeath : MonoBehaviour
+{
+    private Scoreboard _scoreboard;
```

Comment:
I know it sounds weird, but since this is getting filled automatically, make it `public` so we can see it in the editor when we run the game and we can see what scoreboard it "finds" if we later need to debug it